### ISSUE TITLE
Speakerinnen* birthday next Tuesday (and international womens' day)

### ISSUE THREAD

**Issue by Powerglove** [rubynettes/rubynettes#52]
Next tuesday at taz there will be a birthday party for the platform speakerinnen*, that was programmed in the Rails Girls study group Ruby Monstas. I would really like to go there and was wondering if any of you are interested as well? Here's the link:

http://taz.de/tazveranstaltung-zum-Frauentag-0803/!161906/

Maybe we can go on a study group excursion? ;)

**Comment by stefanwille** [comment]
Let's talk about this on Saturday.

**Comment by Powerglove** [comment]
So far I talked to Stefan and Conny and they both like the idea of going to this party. What about the others who are planning to come on Tuesday? @mariaalavez @yuluoluo 
BTW is Sandra already part of this group on Github and can read these messages?

**Comment by mariaalavez** [comment]
Hi all!
I can definitely come! but a bit later. It depends on the day's workload :)

**Comment by stefanwille** [comment]
No, I emailed Sandra before for her GitHub handle, haven't received it yet.

**Comment by stefanwille** [comment]
Okay, since I am still sick, let's do this: You guys go to the SpeakerInnnen party and I will stay in bed.

**Comment by supernoir** [comment]
Get well soon, @stefanwille!

**Comment by stefanwille** [comment]
Heads up to everybody: Today will be **no session at HERE**, the group will go to the SpeakerInnen party.
